lib/postprocessing/image.js

```diff
@@ -261,10 +261,12 @@ async function applyOperations(img, rendition, directories) {
     }
 }
 
-async function applyOutputProperties(img, instructions) {
-    // for reproducible png files
-    if (instructions.fmt === "png") {
+async function applyOutputProperties(img, instructions) {   
+    if (instructions.fmt === "png") { // for reproducible png files
         img.define("png:exclude-chunks=date");
+    } else if (instructions.fmt === "tif" 
```

Comment:
Suggestion: to keep it easier to read and consistent, each `fmt` if condition should be in a separate `if` block like the others. There will be many over time and they will become larger. Makes it easier to read. Going by fmt makes them naturally exclusive. And have comments on their own line right above the statement:

```js
    if (instructions.fmt === "png") {
        // for reproducible png files
        img.define("png:exclude-chunks=date");
    }
    
    if (instructions.fmt === "tif" || instructions.fmt === "tiff") {
        // preserve same compression as initial file (e.g. for png -> tiff)
        img.define("tiff:preserve-compression=true");
    }
```

Reply:
I would actually prefer the if/else option (personal taste, I guess). THis makes the exclusion in-between expression more explicit. This would also avoid retesting all if blocks after the execution path actually went through one.

To finish, if the issue is that you expect those conditions to grow:
1. If it grows, it is going to go in its own function, something like `applyFormatOutputOptions`.
2. `if/else`usage would prevent multiple condition tests.